tests/models/sniff_test.py

```diff
@@ -0,0 +1,84 @@
+# pylint: disable=line-too-long,bad-whitespace,no-self-use
+
+from allennlp.commands import DEFAULT_MODELS
+from allennlp.common.testing import AllenNlpTestCase
+from allennlp.models.archival import load_archive
+from allennlp.service.predictors import Predictor
+
+
+class SniffTest(AllenNlpTestCase):
+
+    def test_config(self):
+        assert set(DEFAULT_MODELS.keys()) == {
+                'machine-comprehension',
+                'semantic-role-labeling',
+                'textual-entailment'
+        }
+
+
+    def test_machine_comprehension(self):
+        predictor = Predictor.from_archive(
```

Comment:
It might be worth modifying the [cache directories](https://github.com/allenai/allennlp/blob/e8a4a9c997a646618465ba42388fff7731ead34b/.travis.yml#L4-L9) in the Travis config to include our file cache dir.  It would speed this up a little.